WordPress/gutenberg
# Makes spacing consistent in zoom out vertical toolbar

## What?
Fixes: #63807 

## Why?
See this [comment](https://github.com/WordPress/gutenberg/issues/63807#issue-2422827049)

## Testing Instructions
1. Enable the zoomed out view
2. Go to any post and try to add a pattern by clicking on the inserter icon.
3. See the vertical toolbar controls and check the spacing between them.

## Screenshots or screencast 
<img width="1470" alt="Screenshot 2024-07-26 at 6 41 57 PM" src="https://github.com/user-attachments/assets/5d672d85-c737-4dd0-a672-0235e94a289d">

Co-authored-by: amitraj2203 <amitraj2203@git.wordpress.org>
Co-authored-by: getdave <get_dave@git.wordpress.org>
Co-authored-by: draganescu <andraganescu@git.wordpress.org>
Co-authored-by: richtabor <richtabor@git.wordpress.org>
